gqueries/general/flexibility/cumulative_curve/all_time_resolved_carriers_demand_inflexible_curve.gql

```diff
@@ -0,0 +1,8 @@
+- query =
+    SUM_CURVES(
+      Q(electricity_demand_inflexible_curve),
+      Q(network_gas_demand_inflexible_curve),
+      Q(heat_network_demand_inflexible_curve),
+      Q(hydrogen_demand_inflexible_curve)
+    )
+- unit = MJ
```

Comment:
Is this unit correct? Same for curves below.

Reply:
No that's not correct, I'll convert it to `- unit = curve` ! Good catch